sync-core/src/main/java/com/cloudant/sync/datastore/AttachmentManager.java

```diff
@@ -159,7 +170,7 @@ protected PreparedAndSavedAttachments prepareAttachments(Collection<? extends At
         return preparedAndSavedAttachments;
     }
 
-    protected void setAttachments(SQLDatabase db,BasicDocumentRevision rev, PreparedAndSavedAttachments preparedAndSavedAttachments) throws IOException {
+    protected void setAttachments(SQLDatabase db,BasicDocumentRevision rev, PreparedAndSavedAttachments preparedAndSavedAttachments) throws AttachmentNotSavedException, AttachmentException {
```

Comment:
AttachmentNotSavedException is reduandant here as it's a subclass of AttachmentException